src/components/HomeContainer/SuceessStories/SuccessStories.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from "react";
+import typography from "../../utility/Typography/Typography.module.css";
+import styles from "./SuccessStories.module.css";
+
+import avatar_Kady from "../../../assets/avatar-kady.jpg";
+import avatar_Aisha from "../../../assets/avatar-aiysha.jpg";
+import avatar_Arthur from "../../../assets/avatar-arthur.jpg";
+
+
+
+const SuccessStory = ({ quote, name, title, avatar, altText }) => {
+  return (
+    <div className={styles.SuccessStories_content}>
+      <p className={typography.body2}>{quote}</p>
+      <h3 className={`${typography.h3} ${styles.SuccessStories_name}`}>{name}</h3>
+      <p className={styles.SuccessStories_caption}>{title}</p>
+      <div className={styles.SuccessStories_image}>
+        <img src={avatar} alt={altText} />
+      </div>
+    </div>
+  );
+};
+
+export const SuccessStories = () => {
+  return (
+    <section className={styles.SuccessStories}>
+      <div className={styles.SuccessStories_wrapper}>
+        <h2 className={`${typography.h2} ${styles.SuccessStories_heading}`}>
+          Delivering real results for top companies. Some of our <span>success stories.</span>
+        </h2>
+
+        <article className={styles.SuccessStories_contents}>
+          <SuccessStory
```

Comment:
We can avoid this repetition by adding all the story details in an object like this 
`const stories = [
  {
    id: 1,
    quote:
      "The team perfectly fit the specialized skill set required. They focused on the most essential features helping us launch the platform eight months faster than planned.",
    personName: "Kady Baker",
    title: "Product Manager at Bookmark",
    avatar: avatar_Kady,
    altText: "Kady Baker",
  },
  {
    id: 2,
    quote:
      "We needed to automate our entire onboarding process. The team came in and built out the whole journey. Since going live, user retention has gone through the roof!",
    personName: "Aiysha Reese",
    title: "Founder of Manage",
    avatar: avatar_Aisha,
    altText: "Aiysha Reese",
  },
  {
    id: 3,
    quote:
      "Amazing. Our team helped us build an app that delivered a new experience for hiring a physio. The launch was an instant success with 100k downloads in the first month.",
    personName: "Arthur Clarke",
    title: "Co-founder of MyPhysio",
    avatar: avatar_Arthur,
    altText: "Arthur Clarke",
  },
];`

Next we can iterate through each of this array elements like the following
`<article className={styles.SuccessStories_contents}>
          {stories.map(({ id, quote, personName, title, avatar, altText }) => {
            return (
              <SuccessStory
                key={id}
                quote={quote}
                personName={personName}
                title={title}
                avatar={avatar}
                altText={altText}
              />
            );
          })}
        </article>`

Reply:
This way the code will be more simpler and easier to add a new story or delete in future, one just needs to work on the constant and not update the JSX anymore